src/sql/workbench/api/node/extHostModelView.ts

```diff
@@ -198,6 +198,13 @@ class ModelBuilderImpl implements sqlops.ModelBuilder {
 		return builder;
 	}
 
+	hyperLink(): sqlops.ComponentBuilder<sqlops.HyperLinkComponent> {
```

Comment:
Hyperlink is all one word so this should probably be `hyperlink`. Likewise `sqlops.HyperlinkComponent`